ang/civicase/CaseList.html

```diff
@@ -25,24 +25,25 @@ <h1 crm-page-title>{{ pageTitle }}</h1>
     <div class="custom-scroll-wrapper">
       <div class="custom-scroll"></div>
     </div>
-    <div class="inner" ng-include="'~/civicase/CaseListTable.html'"></div>
+    <div class="inner" id="caselist-table" ng-include="'~/civicase/CaseListTable.html'"></div>
```

Comment:
The indentation of this file, is not correct on line 24. Please fix it.

Reply:
Done

Reply:
Done